server/request_sendrawtx.go

```diff
@@ -90,6 +107,7 @@ func (r *RpcRequest) handle_sendRawTransaction() {
 	// Check for cancellation-tx
 	if len(r.tx.Data()) <= 2 && txFromLower == strings.ToLower(r.tx.To().Hex()) {
 		requestDone := r.handleCancelTx() // returns true if tx was cancelled at the relay and response has been sent to the user
+		r.ethSendRawTxEntry.IsCancelTx = requestDone
```

Comment:
```suggestion
		r.ethSendRawTxEntry.IsCancelTx = true
```

This is a cancel-tx already, no matter if it has been sent to the relay or not.

Reply:
updated